README.md

```diff
@@ -29,10 +29,19 @@ Please see the following for detailed descriptions of each type:
    ```
    ./generate-go.sh TARGET_PATH
    ```
+   
+### Java
+
+1. Build the go code first (see above) so that all the imports are available
```

Comment:
There probably is a way, but if so it seems like it would deserve another pull request. As it stands isn't it better to have something that works (as opposed to what is currently in master)?